modules/new_model/shared/src/main/scala/observe/model/enums/SystemName.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package observe.model.enums
+
+import cats.Eq
+import cats.derived.*
+import cats.syntax.eq.*
+import observe.model.ParamName
+
+enum SystemName(val system: String) derives Eq:
```

Comment:
Why not `derives Enumerated`?

Reply:
Good idea, I'll implement and remove the parser.